src/Equinox.EventStoreDb/EventStoreDb.fs

```diff
@@ -168,6 +168,13 @@ module private Write =
 
 module private Read =
     open FSharp.Control
+    let private processReadStreamResult (res: EventStoreClient.ReadStreamResult) f onStreamNotFound = task {
```

Comment:
Scanning this right now in web view, I'm not convinced having a separated function makes the overall code comprehension exercise easier - i.e. inlining instead of being DRY might be clearer? (Will look in an IDE later)

Reply:
Yes maybe you're right about. I'm also not happy with the function name tbh

Reply:
I inlined it in https://github.com/jet/equinox/pull/373/commits/740cc4a861f7f851641a7bb05d4e9234877229f1
If you can cherry pick that and/or do similar in your own context, I'll release (or say the word and I can merge yours and apply it as a fixup on master afterwards)

Reply:
Thank you! Cherry picked it into this branch